test/fixtures/mongo_keyfile

```diff
@@ -0,0 +1,16 @@
+PUQGGGC+g1sxoYzKKnX7531pos0WvDCYlx1Ak5ZupX3KvyuU2iVcC9puJJSG06VR
```

Comment:
I'm sure this is fine but I'm confused why we need a keyfile?

Reply:
We want to use the same Mongo setup as in production. The initial idea was to be able to swap this for a staging server / database.

I'm fine with removing it if you're sure we don't need it. I set this up a long time ago and I haven't revisited it since then.

Reply:
🤷‍♂️  I would remove it and use a simple config to set up the test db with. I think we should not necessarily make it easy to swap out staging/production config because that's quite risky actually and you can accomplish the same thing with just adjusting the URI, username and password.